config/200-clusterrole.yaml

```diff
@@ -28,13 +28,13 @@ rules:
     # Controller needs cluster access to all of the CRDs that it is responsible for
     # managing.
   - apiGroups: ["tekton.dev"]
-    resources: ["tasks", "clustertasks", "taskruns", "pipelines", "pipelineruns", "pipelineresources", "runs", "customruns"]
+    resources: ["tasks", "clustertasks", "taskruns", "pipelines", "pipelineruns", "pipelineresources", "runs", "customruns", "verificationpolicies"]
```

Comment:
Does the controller need write access to the policies (outside of status), or can we scope this down to just read-only?

Reply:
I think it should be just read-only